src/components/Widget/transactions.ts

```diff
@@ -1,17 +1,76 @@
 import { TransactionReceipt } from '@ethersproject/abstract-provider'
-import { TransactionEventHandlers } from '@uniswap/widgets'
-import { useMemo } from 'react'
+import { TradeType, Transaction, TransactionEventHandlers, TransactionInfo, TransactionType } from '@uniswap/widgets'
+import { useWeb3React } from '@web3-react/core'
+import { V3_SWAP_DEFAULT_SLIPPAGE } from 'hooks/useAutoSlippageTolerance'
+import { useCallback, useMemo } from 'react'
+import { useTransactionAdder } from 'state/transactions/hooks'
+import {
+  ExactInputSwapTransactionInfo,
+  ExactOutputSwapTransactionInfo,
+  WrapTransactionInfo,
+} from 'state/transactions/types'
+import { useUserSlippageTolerance } from 'state/user/hooks'
+import { currencyId } from 'utils/currencyId'
 
 /** Integrates the Widget's transactions, showing the widget's transactions in the app. */
 export function useSyncWidgetTransactions() {
-  // TODO(jfrankfurt): Integrate widget transactions with app transaction tracking.
+  const { chainId } = useWeb3React()
+  const addTransaction = useTransactionAdder()
+  const [appSlippage] = useUserSlippageTolerance()
+  const allowedSlippage = appSlippage === 'auto' ? V3_SWAP_DEFAULT_SLIPPAGE : appSlippage
```

Comment:
The slippage is actually defined at the time of the transaction by the widget, and the default may not match the interface default. To keep the definition in one reliable place, I think it's best to include this in the trade object emitted by the widget (which will need to be added to it).

Reply:
agree--this default value is what I felt most uncomfortable about in the pr

Reply:
See https://github.com/Uniswap/widgets/pull/187

Reply:
done